demo-extended/templates/deployment.yaml

```diff
@@ -17,12 +18,14 @@ spec:
       containers:
         - name: {{ .Values.name }}
           image: "{{ .Values.general.image }}:{{ .Values.general.version }}"
-          {{- if ((.Values.networking).expose) }}
+          
+          {{- if .Values.networking.expose }}
           ports:
             - name: http
-              containerPort: {{ .Values.networking.port | default 80 }}
+              containerPort: {{ .Values.networking.port }}
```

Comment:
What happens if port is not set? We cant enforce it since exposing is optional

Reply:
Yeah but this template is meant to expose the port specified by the user.